src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcAddToAList.scala.html

```diff
@@ -76,7 +76,8 @@ <h1 class="govuk-heading-xl">
                 visuallyHiddenText = Some(if (language == Cy) s"Dileu’r ${item.name} o’r rhestr" else s"Remove ${item.name} from the list"),
                 href= item.removeUrl
             )
-          ))),
+          ),
+          classes = None)),
```

Comment:
As above, I think the default should be in the viewmodel not the view to follow library convention.